[BUGFIX beta] Fixes #12899

asakusuma opened the pull request:
Ensure transitionTo returns a transition object. Addresses https://github.com/emberjs/ember.js/issues/12899

rwjblue commented:
Sorry for not getting this done quicker, but thank you for tackling it!

rwjblue commented on packages/ember-routing/lib/system/router.js:
```diff
@@ -303,9 +303,26 @@ var EmberRouter = EmberObject.extend(Evented, {
+      containing a mapping of query parameters
+    @return {Transition} the transition object associated with this
+      attempted transition
+    @public
```
Lets land this fix without changing public API (docs here are great, but change `@public` to `@private`), then submit a separate documentation only PR that updates?

asakusuma committed:
[BUGFIX beta] Fixes #12899

Ensure transitionTo returns a transition object. Addresses
https://github.com/emberjs/ember.js/issues/12899

rwjblue commented:
:+1:

asakusuma commented:
No problem, @krisselden got me on the right approach

rwjblue commented:
Yeah, he is awesome like that.  :smiley_cat: 

I need to determine if this bug is in 2.3.1, and backport to release if it is....
